cryptoutil/cryptoutil_test.go

```diff
@@ -0,0 +1,38 @@
+package cryptoutil
+
+import (
+	"testing"
+
+	"github.com/teamwork/test/diff"
+)
+
+const (
+	keyString         = "abcd1234abcd1234"
+	testPlaintextData = "HELLO WORLD"
```

Comment:
Unless I missed something the `testPlaintextData` variable seems unused?

Reply:
You're correct here, but I'm improving the tests right now so it will be 😄